presentation/src/main/java/com/kdjj/presentation/viewmodel/recipedetail/NotificationBroadcastReceiver.kt

```diff
@@ -0,0 +1,26 @@
+package com.kdjj.presentation.viewmodel.recipedetail
+
+import android.app.ActivityManager
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+
+class NotificationBroadcastReceiver : BroadcastReceiver() {
+
+    override fun onReceive(context: Context?, intent: Intent?) {
+        val activityManager = context?.getSystemService(Context.ACTIVITY_SERVICE) as ActivityManager
```

Comment:
```suggestion
        val activityManager = (context?.getSystemService(Context.ACTIVITY_SERVICE) as? ActivityManager) ?: return
```
추가해주시면 조금 더 안전할 것 같아요~